modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -1,5 +1,5 @@
 Issue Tracker Filter Form [Automation Testing]
-1. User can access the page iff they have issue tracker reporter or developer permission
+1. User can access the page if they have issue tracker reporter or developer permission
```

Comment:
```suggestion
1. A user can access the module as long as they have one of the `issue_tracker_reporter` or `issue_tracker_developer` permissions.
```

iff is a short form for "if, and only if,". I agree that it doesn't provide much clarity in this context though so I'm happy to reword the sentence. 

Item 2 should also be reworded.

Reply:
Wow, I didn't know. Thanks @johnsaigle